packages/aloxide-js/src/helpers/validate-network-config.ts

```diff
@@ -3,7 +3,7 @@ export function validateNetworkConfig(config) {
     throw new Error('Missing network config!');
   }
 
-  const requiredKeys = 'name,type,protocol,host,port,chainId,coreToken'.trim().split(',');
+  const requiredKeys = 'name,type,protocol,host,chainId,coreToken'.trim().split(',');
```

Comment:
It is funny that you use trim() and split() here.

Reply:
hmm... we should point the issue when reviewing, "something is funny that..." is an exclamatory phrase and it doesn't show the action I should do here.